templates/android-dot-com/styles.css

```diff
@@ -481,7 +481,8 @@ a img{
     transition: opacity 0.2s cubic-bezier(0.4, 0, 0.2, 1);
   }
 
-  .android-search-box.is-focused ~ .android-mobile-title {
+  .android-search-box.is-focused + .android-navigation-container +
+      .android-mobile-title {
```

Comment:
Looks like you are trying to do a double-sibling-selector... [Could be buggy](https://bugs.webkit.org/show_bug.cgi?id=54373). I don't have an iOS device/setup to verify with, but please run [this testcase](http://codepen.io/Garbee/pen/EVvmog) and verify the background of the text is blue and the color is red.